### ISSUE TITLE
Add customizable keyboard shortcut

### ISSUE THREAD

**Issue by cloudwitch** [mozilla/contain-facebook#613]
- Facebook Container Version: Latest
- Operating System + Version: Clear Linux
- Firefox Version: 73.01
- Other installed Add-ons + Version + Enabled/Disabled-Status:

### Actual behavior
`Ctrl` + `.` opens menu to start a new container.

### Expected behavior
I would like the ability to change the keyboard shortcut as 1password also uses `Ctrl` + `.` to unlock the keyring.

### Steps to reproduce
1. Install 1passwordx
2. Install contain-facebook
3. Press `Ctrl` + `.`, contain-facebook comes up

**Comment by maxxcrawford** [comment]
@pheonix991 Which add-ons do you have installed currently?

**Comment by pheonix991** [comment]
From `about:support`:
```
1Password extension   (desktop app required)
1Password X – Password Manager
AdNauseam
Amazon.com
Bing
Bypass Paywalls
Cisco Webex Extension
Dark Reader
DuckDuckGo
eBay
Facebook Container
Firefox Multi-Account Containers
Google
New Tab Override
OneLogin for Firefox
Privacy Badger
Reddit Enhancement Suite
Terms of Service; Didn’t Read
Text Rewriter
TL;DR - Summarizer
Twitter
Wikipedia (en)
Zoom Scheduler
```

**Comment by maxxcrawford** [comment]
@cloudwitch I think this issue is actually related to [Multi-account Containers[(https://github.com/mozilla/multi-account-containers/issues). I tested a build with ONLY Facebook Container added, and the keyboard short cut doesn't do anything. Please file this issue there. Thanks!
